crates/rspack_testing/src/test_config.rs

```diff
@@ -365,26 +365,29 @@ impl TestConfig {
           rule
             .r#use
             .into_iter()
-            .map(|i| match i.builtin_loader.as_str() {
-              "builtin:sass-loader" => Arc::new(rspack_loader_sass::SassLoader::new(
-                i.options
-                  .map(|options| {
-                    serde_json::from_str::<rspack_loader_sass::SassLoaderOptions>(&options)
-                      .expect("should give a right loader options")
-                  })
-                  .unwrap_or_default(),
-              )) as BoxLoader,
-              "builtin:swc-loader" => Arc::new(rspack_loader_swc::SwcLoader::new(
-                i.options
-                  .map(|options| {
-                    serde_json::from_str::<rspack_loader_swc::SwcLoaderJsOptions>(&options)
-                      .expect("should give a right loader options")
-                  })
-                  .unwrap_or_default(),
-              )) as BoxLoader,
-              _ => panic!("should give a right loader"),
-            })
-            .collect::<Vec<BoxLoader>>(),
+            .map(
+              #[allow(clippy::match_single_binding)]
```

Comment:
Does not support running with builtin loader without Node runtime. (Impl in separate PR)